bundles/core/org.eclipse.smarthome.core.thing.test/src/test/java/org/eclipse/smarthome/core/thing/internal/ChannelStateDescriptionProviderOSGiTest.java

```diff
@@ -153,22 +193,38 @@ public void presentItemStateDescription() {
                 null, "test thing", new Configuration());
         Assert.assertNotNull(thing);
         managedThingProvider.add(thing);
-        final ItemChannelLink link = new ItemChannelLink("TestItem", thing.getChannels().get(0).getUID());
+        Assert.assertNotNull(thing.getChannel("1"));
+        ItemChannelLink link = new ItemChannelLink("TestItem", thing.getChannel("1").getUID());
+        linkRegistry.add(link);
+        Assert.assertNotNull(thing.getChannel("2"));
+        link = new ItemChannelLink("TestItem2", thing.getChannel("2").getUID());
+        linkRegistry.add(link);
+        Assert.assertNotNull(thing.getChannel("3"));
+        link = new ItemChannelLink("TestItem3", thing.getChannel("3").getUID());
+        linkRegistry.add(link);
+        Assert.assertNotNull(thing.getChannel("4"));
+        link = new ItemChannelLink("TestItem4", thing.getChannel("4").getUID());
+        linkRegistry.add(link);
+        Assert.assertNotNull(thing.getChannel("5"));
+        link = new ItemChannelLink("TestItem5", thing.getChannel("5").getUID());
+        linkRegistry.add(link);
+        Assert.assertNotNull(thing.getChannel("6"));
+        link = new ItemChannelLink("TestItem6", thing.getChannel("6").getUID());
         linkRegistry.add(link);
         //
         final Collection<Item> items = itemRegistry.getItems();
         Assert.assertEquals(false, items.isEmpty());
 
-        Item numberItem = null;
-        for (final Item item : items) {
-            if (item.getType().equals("Number")) {
-                numberItem = item;
-                break;
-            }
+        Item item;
+        try {
+            item = itemRegistry.getItem("TestItem");
+        } catch (ItemNotFoundException e) {
```

Comment:
The next assertions in this test method will not run, but as this aspect fails anyway this is not a problem. The other test methods should be executed individually.